2022-03-30 18:04:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FELHASZNÁLÓK" not found; SQL statement:
select * from felhasználók [42102-199]
2022-03-30 18:05:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FELHASZNÁLÓK" not found; SQL statement:
select * from felhasználók.sa [90079-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.readTableFilter(Parser.java:1864)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:2641)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2788)
	at org.h2.command.Parser.parseSelectSub(Parser.java:2636)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:2469)
	at org.h2.command.Parser.parseSelect(Parser.java:2440)
	at org.h2.command.Parser.parsePrepared(Parser.java:814)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-03-30 18:05:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FELHASZNÁLÓK" not found; SQL statement:
select * from FELHASZNÁLÓK [42102-199]
2022-03-30 18:05:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FELHASZNÁLÓK" not found; SQL statement:
select * from FELHASZNÁLÓK.SA [90079-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:573)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.readTableFilter(Parser.java:1864)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:2641)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2788)
	at org.h2.command.Parser.parseSelectSub(Parser.java:2636)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:2469)
	at org.h2.command.Parser.parseSelect(Parser.java:2440)
	at org.h2.command.Parser.parsePrepared(Parser.java:814)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-03-30 18:13:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM INFORMATION_SCHEMA.CATALOGS INFORMATION_SCHEMA.[*]USERS"; SQL statement:
SELECT * FROM INFORMATION_SCHEMA.CATALOGS INFORMATION_SCHEMA.USERS [42000-199]
2022-04-12 20:47:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""CROWD"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 20:47:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""CROWD"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 21:19:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""CROWD"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 21:19:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""CROWD"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 21:34:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""KM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 21:34:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""KM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 22:06:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """EMAIL"" VARCHAR(255)": "STRINGDECODE('Sorsz\u00e1m\tN\u00e9v\tE-Mail\tLakc\u00edm\tTelefon\tJelsz\u00f3 1\tFarag\u00f3 Lajos\tflajos75@freemail.hu\t3358... (1983)"; SQL statement:
INSERT INTO "PUBLIC"."USER"("ID","ADRESS","NAME","PASSWORD","PHONE","USERNAME","EMAIL")VALUES(?,?,?,?,?,?,?) [22001-199]
2022-04-12 22:06:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """EMAIL"" VARCHAR(255)": "STRINGDECODE('Sorsz\u00e1m\tN\u00e9v\tE-Mail\tLakc\u00edm\tTelefon\tJelsz\u00f3 1\tFarag\u00f3 Lajos\tflajos75@freemail.hu\t3358... (1983)"; SQL statement:
INSERT INTO "PUBLIC"."USER"("ID","ADRESS","NAME","PASSWORD","PHONE","USERNAME","EMAIL")VALUES(?,?,?,?,?,?,?) [22001-199]
2022-04-12 22:30:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""ROLLING"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 22:30:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""ROLLING"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 22:34:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""CROWD"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-12 22:34:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'' (CAR: ""CROWD"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."CAR"("ID","BRAND","COLOUR","CROWD","FUEL","GEARBOX","KM","LOOK","PERFORMANCE","PERSON","PRICE","ROLLING","TYPE","YEAR")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14) [22018-199]
2022-04-16 16:56:45 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:242)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:182)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:299)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:782)
	at org.h2.server.web.WebApp.login(WebApp.java:1004)
	at org.h2.server.web.WebApp.process(WebApp.java:219)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 15 more
2022-04-16 17:40:44 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:242)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:182)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:299)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:782)
	at org.h2.server.web.WebApp.login(WebApp.java:1004)
	at org.h2.server.web.WebApp.process(WebApp.java:219)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 15 more
